assets/app/util/githubApi.js

```diff
@@ -1,25 +1,31 @@
 import http from 'axios';
+import fetch from './fetch';
 
 import store from '../store';
 import { encodeB64, decodeB64 } from './encoding';
 import errorActions from '../actions/errorActions';
 
 const API = 'https://api.github.com';
 
+// Thinking this function should not live here
+// Service should be agnostic as to what the store is.
+// Access token should be passed in to the functions
```

Comment:
I'm fine with passing in the token but since it will likely be the same user and same token it might be nice to only need to use it once. Maybe something like a factory function where it returns an object that has access to an internal token?

Either way works for me in the end though.